<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidgettext</class>
 <widget class="QWidget" name="LogWidgettext">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cb_debug">
       <property name="text">
        <string>DEBUG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_info">
       <property name="text">
        <string>INFO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_warn">
       <property name="text">
        <string>WARN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_critical">
       <property name="text">
        <string>CRITICAL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
